cmd/cost/aws.go

```diff
@@ -0,0 +1,51 @@
+package cost
+
+import (
+	"github.com/spf13/cobra"
+
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/costexplorer"
+	"github.com/aws/aws-sdk-go/service/organizations"
+)
+
+// awsCmd represents the aws command
+var awsCmd = &cobra.Command{
+	Use:   "aws",
+	Short: "A brief description of your command",
+	Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("aws called")
+	},
+}
+
+func init() {
+	awsCmd.AddCommand(getCmd)
+	awsCmd.AddCommand(createCmd)
+	awsCmd.AddCommand(reconcileCmd)
+}
+
+func initAWSClients() (*organizations.Organizations, *costexplorer.CostExplorer) {
```

Comment:
There is already two functions to initialize AWS Clients https://github.com/openshift/osd-utils-cli/blob/master/pkg/provider/aws/client.go here. You can add costexplorer and organizations clients there and add more interfaces.